lib/components/inputs/SearchField/SearchField.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react'
+import TextBox from '../TextBox'
+import { EMPTY_STRING } from '../../../consts'
+import { useDebounce } from '../../../hooks'
+
+import './searchField.scss'
+
+export interface SearchFieldProps {
+  label: string
+  onValueUpdate: (value: string) => void
+  debounceDelay?: number
+  shouldUpdateTerm?: boolean
+}
+
+function SearchField(props: SearchFieldProps) {
+  const {
+    debounceDelay = 700,
```

Comment:
Let's use some constant, either for default delay in `debounce` hook itself, or place it here if it is specific case